packages/expo-av/src/__stories__/Video.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+
+import VideoPlayer from './components/VideoPlayer';
+
+export const VideoHttpPlayer = () => (
+  <VideoPlayer
+    sources={[
+      { uri: 'http://d23dyxeqlo5psv.cloudfront.net/big_buck_bunny.mp4' },
+      { uri: 'http://techslides.com/demos/sample-videos/small.mp4' },
+      { uri: 'http://qthttp.apple.com.edgesuite.net/1010qwoeiuryfg/sl.m3u8' },
+    ]}
+  />
+);
+
+VideoHttpPlayer.storyConfig = {
+  name: 'Play Remote Videos',
+};
+
+export const LocalAssetPlayer = () => (
+  <VideoPlayer
+    sources={[require('./assets/videos/ace.mp4'), require('./assets/videos/star.mp4')]}
+  />
+);
+
+LocalAssetPlayer.storyConfig = {
+  name: 'Play Local Videos',
+};
+
+export default {
+  title: 'Video',
+};
```

Comment:
I image we can deliver more elegant top-level/default export from each file like this:
```suggestion
export default createStory({
  title: 'Video',
  components: {
    VideoHttpPlater,
    LocalAssetsPlayer,
  },
});
```
This way we have more control over what's exported and what processing is applied in-between exporting and consuming the components 😉 
What would you say? Possibly the current syntax is based on the `storybook`'s syntax/mechanisms?

Reply:
yea you're right about this - its based on the component story format: https://storybook.js.org/docs/react/api/csf which exports just components and some configs 

I like this idea though, I'll think about it some more!